### ISSUE TITLE
$guid parameter reported as not found in editor even though it executes/behaves correctly

### ISSUE THREAD

**Issue by patrickhammond** [Huachao/vscode-restclient#551]
- REST Client Version: 0.23.2
- VSCode Version: 1.43.1
- OS Version: macOS Catalina 10.15.3

Steps to Reproduce:
1. Create a file named `example.rest` with the following content:
```
GET {{baseUrl}}/api/v1/app/config
Accept: application/json; charset=UTF-8
X-Request-ID: {{$guid}}
```
2. Notice that it reports `$guid is not found` even though the request will execute correctly.

![Screen Shot 2020-03-19 at 5 07 41 PM](https://user-images.githubusercontent.com/424944/77115141-2a480d80-6a04-11ea-900a-1d957dd1d976.png)

![Screen Shot 2020-03-19 at 5 08 48 PM](https://user-images.githubusercontent.com/424944/77115232-5b284280-6a04-11ea-87ff-d3e12652e0a5.png)

**Comment by Huachao** [comment]
@patrickhammond nice catch, this is a regression issue introduced in version 0.23.2, I have fixed it and will publish it in the next release.

**Comment by Huachao** [comment]
@patrickhammond this should be fixed in the latest version 0.24.0, and you can verify it.
